### PR TITLE
Update `area_create` description in `PhysicsServer2D/3D` to clear up possible confusions

### PR DESCRIPTION
**Pull request by TheKiromen**
I'm a newbie who never worked with game engine before so apologies in advance if this is incorrect.

I spent way too long trying to figure out why collisions with areas created from Area2D nodes were working but not between areas created using PhysicsServer2D.area_create().

From my testing it looks like by default the `monitorable` property is set to false, which can be confusing since there is no easy way to check it (no `area_get_monitorable()` method for example). Especially since areas created with Area2D nodes have this enabled by default.

To reduce chances other people will be confused like me I updated the description of the method to explicitly state that. 

I didn't mention `monitoring` property since areas are always monitoring, and disabling it in Area2D node just means setting empty callback. If you think there may be value in adding this as well, feel free to suggest changes.

### CODE REVIEW

**Comment by rburing** [comment]
Looks good to me. Please copy the whole description to the method with the same name in `PhysicsServer3D` (and replace `2D` by `3D`), and I can approve the PR.

**Review by rburing — DISMISSED** [review]

File: doc/classes/PhysicsServer3D.xml [hunk 1]
```diff
@@ -44,7 +44,8 @@
 		<method name="area_create">
 			<return type="RID" />
 			<description>
-				Creates an [Area3D].
+				Creates an [Area3D] object in the physics server, and returns the [RID] that identifies it. The default settings for the created area include a collision layer and mask set to [code]1[/code], and [code]monitorable[/code] set to [code]false[/code].
```
Comment (AThousandShips) [inline doc/classes/PhysicsServer3D.xml:47]:
```suggestion
				Creates a 3D area object in the physics server, and returns the [RID] that identifies it. The default settings for the created area include a collision layer and mask set to [code]1[/code], and [code]monitorable[/code] set to [code]false[/code].
```
To make it a bit more clear that this area *isn't* an `Area3D` and match the 2D text better

**Review by AThousandShips — DISMISSED** [review]

File: doc/classes/PhysicsServer3D.xml [hunk 2]
```diff
@@ -44,7 +44,8 @@
 		<method name="area_create">
 			<return type="RID" />
 			<description>
-				Creates an [Area3D].
+				Creates a Area3D object in the physics server, and returns the [RID] that identifies it. The default settings for the created area include a collision layer and mask set to [code]1[/code], and [code]monitorable[/code] set to [code]false[/code].
```
Comment (AThousandShips) [inline doc/classes/PhysicsServer3D.xml:47]:
```suggestion
				Creates a 3D area object in the physics server, and returns the [RID] that identifies it. The default settings for the created area include a collision layer and mask set to [code]1[/code], and [code]monitorable[/code] set to [code]false[/code].
```

File: doc/classes/PhysicsServer2D.xml [hunk 3]
```diff
@@ -369,7 +370,8 @@
 		<method name="body_create">
 			<return type="RID" />
 			<description>
-				Creates a 2D body object in the physics server, and returns the [RID] that identifies it. Use [method body_add_shape] to add shapes to it, use [method body_set_state] to set its transform, and use [method body_set_space] to add the body to a space.
+				Creates a 2D body object in the physics server, and returns the [RID] that identifies it. The default settings for the created body include a collision layer and mask set to [code]1[/code], and [enum BodyMode] is set to [code]BODY_MODE_RIGID[/code].
```
Comment (AThousandShips) [inline doc/classes/PhysicsServer2D.xml:373]: I'd restore this, it's outside the scope of this PR and should be reviewed separately, let's not drift here

File: doc/classes/PhysicsServer3D.xml [hunk 4]
```diff
@@ -351,6 +352,8 @@
 		<method name="body_create">
 			<return type="RID" />
 			<description>
+				Creates a 3D body object in the physics server, and returns the [RID] that identifies it. The default settings for the created body include a collision layer and mask set to [code]1[/code], and [enum BodyMode] is set to [code]BODY_MODE_RIGID[/code].
```
Comment (AThousandShips) [inline doc/classes/PhysicsServer3D.xml:355]: Same here

**Review by rburing — APPROVED** [review]
Looks good to me.

**Review by AThousandShips — APPROVED** [review]

**Comment by AThousandShips** [comment]
Please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

**Comment by AThousandShips** [comment]
Please clean up the commit message so it's just the necessary information, like the PR title, excluding all the "Update X" messages

**Comment by TheKiromen** [comment]
I think I managed to do it, should I create a separate PR for body_create changes?

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**